.github/workflows/ci_skipped.yaml

```diff
@@ -2,15 +2,11 @@ name: CI Skipped
 on:
   pull_request:
     branches: ["**"]
-    paths-ignore:
-      - '**/*.rs'
-      - '**/*.toml'
-      - '**/*.yaml'
-      - '**/*.sh'
-      - '**/*.json'
-      - '**/*.rlp'
-      - 'Dockefile*'
-
+    paths:
+      - 'README.md'
+      - 'LICENSE'
+      - "**/README.md"
+      - "**/docs/**"
```

Comment:
This revert a change that spawned CI skipped checks on PRs that didn't need them.